test/testcafe/spec/DeviceSSOExtensionView_spec.js

```diff
@@ -157,16 +160,21 @@ test
   });
 
 test
-  .requestHooks(logger, verifyErrorMock)('hideds the spinner when authenticator verification fails', async t => {
+  .requestHooks(logger, verifyErrorMock)('hides the spinner when authenticator verification fails', async t => {
     const ssoExtensionPage = new BasePageObject(t);
     await ssoExtensionPage.navigateToPage();
-    await t.expect(Selector('.spinner').getStyleProperty('display')).eql('none');
+    await ssoExtensionPage.formExists();
+    await ssoExtensionPage.form.waitForErrorBox();
```

Comment:
wait for the error page to load before checking if the spinner exists

Reply:
Is this only needed for v3? Or does it also apply to v2?

Reply:
actually this only applies to v3 since we dont need to check the spinner only checking if the error shows up